quickstarts/router-ext/README.md

```diff
@@ -1,8 +1,7 @@
 ## IBM Business Automation Manager Open Editions SmartRouter Extension Example
 
-Note: this is the development branch, the target images might not be available here, instead you can look at the [release branch](https://github.com/jboss-container-images/rhpam-7-openshift-image/tree/7.13.x-blue/quickstarts/router-ext)
 
-Use this quick start guide with the [IBM BAMOE SmartRouter](https://github.com/jboss-container-images/rhpam-7-openshift-image/tree/main/smartrouter) image.
+Use this quick start guide with the [RHPAM SmartRouter](https://github.com/jboss-container-images/rhpam-7-openshift-image/tree/8.0.3.GA/smartrouter) image.
```

Comment:
Please update the link to https://github.com/jboss-container-images/rhpam-7-openshift-image/tree/7.13.x-blue/smartrouter

I guess instead of `RHPAM SmartRouter` it should keep as `IBM BAMOE SmartRouter` as it was before?

Reply:
It should, thanks.